### ISSUE TITLE
[Combobox with filter] set to defaultValue when no results 

### ISSUE THREAD

**Issue by markortiz** [jquense/react-widgets#334]
Hi i really like this widget!

Is it possible to select `defaultValue` or the the first `item` in the `data` when there is no results found if `filter` in enable?

I want to enable `filter='contain'` and also the `suggestions` at the same time. 
And if there is no results found it must select the `defaultValue` 
or have the same behavior when `filter=false` and `suggestions=true` it will select the first `item` in the `data`

**Comment by jquense** [comment]
Hi there,

There is no way to auto select something when there is no filter result, however you can use `value` and `onChange` to finely control what value is set on the component, In general though defaultValue` is not really a "default" value, its an Initial value. It's named defaultValue to be consistent with react inputs, who (I think) picked the wrong name a while ago :P

**Comment by markortiz** [comment]
Thanks @jquense!

What I just did is I added `onBlur` function that will check if the `input data` is in the `list` then if no results found I will change the `value` to `defaultValue`.

However I find it redundant to recheck if `input data` is in or not in the list.
Is there a way where I can check if the `emptyFilter` is showed? 
Or can I insert a function to `emptyFilter` so i can call my own function that will handle `empty` results.
